test-cloud/test-cloud-remote/src/test/filtered-resources/kjars-sources/rule-project/src/main/resources/rules.drl

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+import java.util.List;
+
+global List list
+
+rule "Fast first rule in first activation group" @Propagation(IMMEDIATE)
+activation-group "first-group"
+    salience 10
+when
+then
+    list.add("Fast first rule in first activation group executed from " + System.getProperty("org.kie.server.location", "unknow location"));
+end
+
+rule "Fast second rule in first activation group" @Propagation(IMMEDIATE)
+activation-group "first-group"
+    salience 5
+when
+then
+    list.add("Fast second rule in first activation group executed from " + System.getProperty("org.kie.server.location", "unknow location"));
+end
+
+rule "Slow rule without activation group" @Propagation(IMMEDIATE)
+    salience 2
+when
+then
+    try {
+        Thread.sleep(60000);
+    } catch (Exception e) {
+        System.out.println(e);
```

Comment:
Is it intended to catch just InterruptedException here? Please don't make catch statements too broad.
Maybe would be better to re-throw caught exception as RuntimeException.